homeassistant/components/alarmdecoder/alarm_control_panel.py

```diff
@@ -66,7 +66,7 @@ def alarm_keypress_handler(service):
 class AlarmDecoderAlarmPanel(AlarmControlPanel):
     """Representation of an AlarmDecoder-based alarm panel."""
 
-    def __init__(self):
+    def __init__(self, auto_bypass=False):
```

Comment:
A default value is redundant here since you already have a default value in the respective voluptuous schema

Reply:
Is the default value coming from the voluptuous schema applied even when the parameter is not included in the configuration at all (as it's optional)?

Reply:
Yes, the default value is applied even when it is optional.

Reply:
Done